hydromt/cli/main.py

```diff
@@ -440,14 +440,15 @@ def export(
     Example usage:
     --------------
 
-    export the data of in a single source, in a pertcular region
-    hydromt export -r "{'subbasin': [-7.24, 62.09], 'uparea': 50}" -t era5_hourly -d ../hydromt/data/catalogs/artifact_data.yml .
+    export the data of in a single source, in a particular region, in a particular time range
+    hydromt export -r "{'bbox': [4.6891,52.9750,4.9576,53.1994]}" -s era5_hourly -d ../hydromt/data/catalogs/artifact_data.yml -t '["2010-01-01", "2022-12-31"]' path/to/output_dir
+    note to sam: only support BBOX and GEOM
 
-    export all data of in a single source
-    hydromt export --dd -t era5_hourly .
+    export all data of in a single source of the deltares data catalog
+    hydromt export --dd -s era5_hourly path/to/output_dir
```

Comment:
I think we want to deprecate the --dd flag:
```suggestion
    hydromt export -d deltares_data -s era5_hourly path/to/output_dir
```